docs/visualization/plotting-overview.ipynb

```diff
@@ -123,7 +135,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plot(d['y', 4:7])"
+    "plot(d['y', 4 * sc.units.us : 7 * sc.units.us])"
```

Comment:
Any reason for using value-based indexing here? I feel it makes the plotting docs more confusing, since not every user will use value-based indexing (or have read that docs page already).

Reply:
Since the plots have sorted coordinates, it seemed like a good opportunity to work with coordinate values directly rather than indices. I think you are probably right about this though, it forces us to introduces/educate on sections that the reader might not be familiar with.